server/src/main/scala/net/wiringbits/services/UsersService.scala

```diff
@@ -83,6 +89,35 @@ class UsersService @Inject() (
     } yield Login.Response(user.id, user.name, user.email, token)
   }
 
+  def forgotPassword(request: ForgotPassword.Request): Future[ForgotPassword.Response] = {
+    for {
+      _ <- validateCaptcha(request.captcha)
+      userMaybe <- repository.find(request.email)
+      _ = userMaybe.foreach(user => {
```

Comment:
`.foreach { user => ... }` works, still, this is bad because you are discarding the result while we care if the email was sent, you can do this:

```
user = userMaybe.getOrElse(throw new RuntimeException("Email not registered"))
_ <- enforeVerifiedUser(user)
_ <- emailApi.sendEmail(...)
```

It is ideal to not fail when the user isn't registered but mention something like `If your account is registered you should get an email to reset your password`, which we can do in another PR